graphviz2drawio/models/SVG.py

```
@@ -1,13 +1,38 @@
 from xml.etree.ElementTree import Element
+import xml.etree.ElementTree as ET
+import re
 
 NS_SVG = "{http://www.w3.org/2000/svg}"
 
 
+def parse_svg_path(path_data):
```

Ruff (0.8.0)

Missing return type annotation for public function `parse_svg_path`

(ANN201)